src/types.ts

```diff
@@ -122,7 +128,7 @@ export interface SamlOptions extends Partial<SamlSigningOptions>, MandatorySamlO
   signMetadata?: boolean;
 
   // InResponseTo Validation
-  validateInResponseTo: boolean;
```

Comment:
This is going to land in a semver-major release, let's just do the break here get rid of the boolean; that will keep other parts of the code cleaner.

Reply:
I was hoping to see it in a minor version in order for it to be available faster through `passport-saml`  (because, in the end, I need this).

Reply:
`node-saml` will only be released as a 4.0 release. `passport-saml` will be released with a breaking 4.0 based on `node-saml`. The currently `passport-saml`, 3.x, does not depend on `node-saml`. So, if you need it sooner, then this PR will have to land against the 3.x branch of `passport-saml`.

Having said all that, we are shooting to release v4 along with Node 14 in a few weeks.

Reply:
Ok then, I'm making the change